dipu/torch_dipu/csrc_dipu/runtime/distributed/ProcessGroupDICL.h

```diff
@@ -167,6 +167,9 @@ class DIPU_API ProcessGroupDICL : public Backend {
     return DICL_BACKEND_NAME;
   }
 
+  // NOLINTNEXTLINE(readability-const-return-type) just follow parent class.
+  std::string getCommName(at::DeviceIndex device_index);
```

Comment:
```suggestion
  std::string getCommName(at::DeviceIndex device_index) const;
```
整串调用都可以加上 const，包括 py 层参数

Reply:
getCommName 里调用了 getDICLComms，应该不能是 const 的